mappings/net/minecraft/client/gui/screen/world/WorldEntryListWidget.mapping

```diff
@@ -1,72 +1,84 @@
+	FIELD f_gmmkqxli SNAPSHOT_TOOLTIP_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_hkspkame ERROR_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_ijizppsv HIGHLIGHTED_ERROR_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_iryfsjin HIGHLIGHTED_MARKED_JOIN_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_iudprwiz JOIN_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_iwchirdh loadingEntry Lnet/minecraft/unmapped/C_wwgrvynj$C_chlqbstq;
 	FIELD f_lejfmpml query Ljava/lang/String;
 	FIELD f_mbtquola parent Lnet/minecraft/unmapped/C_yrypplpi;
-	FIELD f_neodhkus FROM_NEWER_VERSION_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_neodhkus FROM_NEWER_VERSION_TOOLTIP_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_nhmjctua LOGGER Lorg/slf4j/Logger;
+	FIELD f_okvxmmxk EXPERIMENTAL Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_rtudrfdo LOCKED Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_snbxmdyx WARNING_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_xjlsplba UNKNOWN_SERVER_LOCATION Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_xshrcksa MARKED_JOIN_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 	METHOD <init> (Lnet/minecraft/unmapped/C_yrypplpi;Lnet/minecraft/unmapped/C_ayfeobid;IIIIILjava/lang/String;Lnet/minecraft/unmapped/C_wwgrvynj;)V
 		ARG 1 parent
 		ARG 2 client
 		ARG 3 screenWidth
 		ARG 4 screenHeight
 		ARG 5 top
 		ARG 6 bottom
-		ARG 7 itemHeight
+		ARG 7 entryHeight
 		ARG 8 query
-		ARG 9 previousList
+		ARG 9 prev
 	METHOD m_adqicvjj narrateScreen ()V
-	METHOD m_bhyykuuh worldNameMatches (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
+	METHOD m_bhyykuuh summaryMatches (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
+		COMMENT Whether the given {@code summary} matches the given {@code query}.
 		ARG 1 query
 		ARG 2 summary
-	METHOD m_egkdspch finalizeWorlds ()Ljava/util/List;
+	METHOD m_egkdspch finalizeSummaryList ()Ljava/util/List;
 	METHOD m_fapsoaqi getSelectedAsOptional ()Ljava/util/Optional;
 	METHOD m_fftsxclm (Ljava/lang/Throwable;)Ljava/util/List;
 		ARG 1 throwable
-	METHOD m_gojurhkm generateListFuture ()Ljava/util/concurrent/CompletableFuture;
-	METHOD m_hexfpxkc filterWorlds (Ljava/util/List;)V
-		ARG 1 worlds
+	METHOD m_gojurhkm createSummaryListFuture ()Ljava/util/concurrent/CompletableFuture;
+	METHOD m_hexfpxkc filterEntries (Ljava/util/List;)V
+		ARG 1 summaries
 	METHOD m_irbmeiln resetEntries ()V
 	METHOD m_jtfjknzc filter (Ljava/lang/String;Ljava/util/List;)V
 		ARG 1 query
-		ARG 2 levels
+		ARG 2 summaries
 	METHOD m_lmtvmxlw getParent ()Lnet/minecraft/unmapped/C_yrypplpi;
 	METHOD m_pfgywwta filter (Ljava/lang/String;)V
 		ARG 1 query
-	METHOD m_qcspbwca failedLoadingWorld (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
+	METHOD m_qcspbwca openUnableToLoadErrorScreen (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 1 reason
-	METHOD m_qscakuku regenerateList ()V
+	METHOD m_qscakuku recreateSummaryList ()V
 	CLASS C_chlqbstq LoadingWorldEntry
```

Comment:
```suggestion
 	CLASS C_chlqbstq LoadingEntry
```
This is an entry for displaying "loading world".  I think it is clearer to name it `LoadingEntry` to not be confused with some variant of `WorldEntry`.